CMakeLists.txt

```diff
@@ -419,7 +430,13 @@ target_link_libraries(Common Ext::Snappy)
 
 if(WIN32)
 	include_directories(dx9sdk/Include)
-	target_link_libraries(Common winmm d3d9 dxguid dsound)
+   include_directories(dx9sdk/Include/DX11)
+   if(MINGW)
+      set(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> -O coff <INCLUDES> <SOURCE> <OBJECT>")
+      # required when using the dx9sdk include paths
+      add_definitions(-include ${CMAKE_CURRENT_SOURCE_DIR}/Windows/mingw_defines.h)
+      add_definitions(-D_WIN32_WINNT=0x0601)
```

Comment:
Hmm, can we get this from stdafx.h?  Also we seem to use 0x600 elsewhere - could we use the same here?

-[Unknown]

Reply:
0x601 is the minimum required version to compile some files (touch API usage mostly).

Reply:
Ah - that's a good catch, we should change that, then.

-[Unknown]